packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -360,32 +350,48 @@ interface ApiExtractionOptions {
   outputDir: string;
   isLocalBuild: boolean;
   tsconfigFilePath: string;
+  allowWarnings: boolean | string[];
+  omitMessages?: string[];
 }
 
 export async function runApiExtraction({
   packageDirs,
   outputDir,
   isLocalBuild,
   tsconfigFilePath,
+  allowWarnings,
+  omitMessages = [],
 }: ApiExtractionOptions) {
   await fs.remove(outputDir);
 
   const entryPoints = packageDirs.map(packageDir => {
     return resolvePath(
-      process.cwd(),
+      cliPaths.targetRoot,
       `./dist-types/${packageDir}/src/index.d.ts`,
     );
   });
 
   let compilerState: CompilerState | undefined = undefined;
 
+  const allowWarningPkg = Array.isArray(allowWarnings) ? allowWarnings : [];
+
+  const messagesConf: { [key: string]: { logLevel: string } } = {};
+  for (const messageCode of omitMessages) {
+    messagesConf[messageCode] = {
+      logLevel: 'none',
+    };
+  }
```

Comment:
I seem to remember that there might be something about this that might not work as you expect. Do the messages still show up inlined in the files for example?

Reply:
I've tested with some messages and seems to work, but I will double check it

Reply:
I tested it again with a couple of codes and seems to work... maybe the problem was related to the `compilerMessageReporting` instead?